celery/app/trace.py

```diff
@@ -378,7 +396,13 @@ def on_error(request, exc, state=FAILURE, call_errbacks=True):
         )
         return I, R, I.state, I.retval
 
-    def trace_task(uuid, args, kwargs, request=None):
+    def trace_task(
+            uuid: str,
+            args: Sequence[Any],
+            kwargs: dict[str, Any],
+            request: Optional[dict[str, Any]] = None) -> trace_ok_t:
```

Comment:
Again, I think you need to use `Dict` here:

```suggestion
            kwargs: Dict[str, Any],
            request: Optional[Dict[str, Any]] = None) -> trace_ok_t:
```